Create Organization Readme.md (#18)
Unleash/.github: profile/README.md

> ![Unleash_GitHub_Banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)

Reviewer: ```suggestion
![Unleash GitHub banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
```

<p align="center">
<b>The Open Source Feature Flagging Service</b>
</p>

> ![Unleash_GitHub_Mocks (2)](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)

Reviewer: Hmm, I'm not sure how best to describe this image. Something like this, perhaps?

```suggestion
![Unleash marketing image showing "Flexible segmentation", "Feature toggles", "Projects" and that Unleash integrates with many programming languages.](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
```

> <!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->

Reviewer: Any reason this is commented out? Could we just remove it?

Author: We can, I was testing it out so I probably missed it during the final PR.

> **Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases, and, in the beginning, we were just a few contributors. Together we built a foundation for the amazing community that you see today. 🫂

Reviewer: Ever since being told by my (Scottish) English teacher in high school that "in the beginning" is very specific to the book of Genesis and that you should always use "at the beginning" if you're not invoking religious imagery, I've tried to avoid it. I did some research now, and from what I can tell, it definitely still carries with it some of that meaning, but it's also not a 100%, so I'm not gonna be completely opposed 🤷🏼 

Also: "We wanted to fill an essential space in the development stack" Did we? I thought Ivar was just solving a need he had himself?

Regardless, I think we can re-jig these sentences some. How about this?

```suggestion
**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases. At first, we were just a few contributors, but together we built a foundation for the amazing community that you see today. 🫂
```

> We love that our community is able to work with our product with all the freedom you could ever want. When you can just test, experiment, and release confidently, you'll end up with a well-rounded product with a ton of awesome interpretations. We're proud of you guys. 🙏

Reviewer 2: I'm not quite sure what interpretations means here?

Author: Interpretations here would mean, to each their own. Calling Unleash as a versatile product to be used anywhere in their stack.

> We love that our community is able to work with our product with all the freedom you could ever want. When you can just test, experiment, and release confidently, you'll end up with a well-rounded product with a ton of awesome interpretations. We're proud of you guys. 🙏

Reviewer: A few minor suggestions on this paragraph:

I think we should avoid gender-specific terminology where possible. While "guys" is often used in a gender-neutral way, it's also subject to heavy debate. Consider other options where applicable or just drop it completely.

I'm also with Simon on the use of "interpretations" here: it's a little unclear. Could we 

I'm also confused about who "you" are in this sentence. It both appears to refer to the general, unspecified "you" (which even sounds a bit like Unleash) **and** to the community.

I'd suggest rewording this a bit. I've take a stab at it here, but feel free to discard this if you have any better ideas ☺️ 

```suggestion
We love that you — our community — have the freedom to work with our product in any way you want. Together we can test, experiment, and create a well-rounded product that covers a ton of different use cases. We're super proud of you 🙏
```

Reviewer 2: I like this! I spent too long thinking about wording I liked here and came up with nothing, really nice suggestion, sir

Author: This is definitely more readable, and puts everything in context. I don't think I can do better than this.

That's enough about us. Have fun. Be kind. Let's **Unleash** great software together. 💪

> [📈 See our roadmap →](https://github.com/orgs/Unleash/projects/5)

## 🚀 Learn about Unleash
Create with Freedom, Release With Confidence.

[Main Open Source Repo](https://github.com/Unleash/unleash) - Unleash's Open Source codebase. 🖥️ </br>
[Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ </br>
[Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚 </br>
[How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋 </br>

> [Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js.</br>

Reviewer: Wait, does this use Next.js? I wasn't aware of that 😅

Author: I believe it does, it's a small detail we can update it later.

[Feature Flagging - Why and How to Add to Your Software](https://youtu.be/-yHZ9uLVSp4) - Need something visual to get started? Here's something to watch. 📹 </br>

## ✨ Connect with Us</br>
Get involved with a vibrant community of users and contributors. Check out what others have built with Unleash, start learning by going through Unleash How-to Guides or get involved as a contributor.

> [Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo 🌟 </br>

Reviewer 2: ```suggestion
[Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo. 🌟 </br>
```

> [Main Open Source Repo](https://github.com/Unleash/unleash) - Unleash's Open Source codebase. 🖥️ </br>
> [Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ </br>
> [Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚 </br>
> [How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋 </br>
> [Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js.</br>
> [Feature Flagging - Why and How to Add to Your Software](https://youtu.be/-yHZ9uLVSp4) - Need something visual to get started? Here's something to watch. 📹 </br>
> 
> ## ✨ Connect with Us</br>
> Get involved with a vibrant community of users and contributors. Check out what others have built with Unleash, start learning by going through Unleash How-to Guides or get involved as a contributor.
> 
> [Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo. 🌟 </br>
> [GitHub Discussions](https://github.com/Unleash/unleash/discussions) - Place to discuss, ask questions and find answers, show your Unleash project and get feedback. 🗣️ </br>
> [Slack](https://unleash-community.slack.com/) -  Come chat with the Unleash community including the core team. 💬 </br>
> [Community Homepage](https://www.getunleash.io/unleash-community) - Check out the ways you can be a valuable member of the community. 🛰️ </br>
> <!-- [Events at Unleash]() - Learn more about new Events for the Unleash Community. 🎪 </br> -->
> Contributing Guidelines coming soon. ⚠️ </br>

Reviewer: Any reason you're using regular sentences instead of a list here? It looks like a list to me, so I think the right way to do this would be to use a list. It would also be helpful for people using assistive tech to read this, so that it's correctly called out as a list 💁🏼 

I'd usually also remove comments, but this seems more like a placeholder for now?

```suggestion
- [Main Open Source Repo](https://github.com/Unleash/unleash) - Unleash's Open Source codebase. 🖥️
- [Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ 
- [Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚
- [How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋
- [Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js
- [Feature Flagging - Why and How to Add to Your Software](https://youtu.be/-yHZ9uLVSp4) - Need something visual to get started? Here's something to watch. 📹

## ✨ Connect with Us

Get involved with a vibrant community of users and contributors. Check out what others have built with Unleash, start learning by going through Unleash How-to Guides or get involved as a contributor.

- [Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo. 🌟
- [GitHub Discussions](https://github.com/Unleash/unleash/discussions) - Place to discuss, ask questions and find answers, show your Unleash project and get feedback. 🗣️
- [Slack](https://unleash-community.slack.com/) -  Come chat with the Unleash community including the core team. 💬 
- [Community Homepage](https://www.getunleash.io/unleash-community) - Check out the ways you can be a valuable member of the community. 🛰️ 
<!-- - [Events at Unleash]() - Learn more about new Events for the Unleash Community. 🎪 -->
- Contributing Guidelines coming soon. ⚠️
```

Author: I don't have a strong preference here, to be honest. Let's move forward with lists.

The comment is a placeholder, yes!

> We can't wait to join you on this journey 🫶

Reviewer: Are we joining them or are they joining us? I feel like it's the other way around, but I'm open to hearing opposing views on this 😄 

```suggestion
We can't wait for you to join us on this journey 🫶
```

Reviewer 2: Same, I think Thomas' suggestion here reads better but also open to opposing views

![Unleash GitHub banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)

<p align="center">
<b>The Open Source Feature Flagging Service</b>
</p>

![Unleash marketing image showing "Flexible segmentation", "Feature toggles", "Projects" and that Unleash integrates with many programming languages.](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)

> 
> 
> <!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->

Reviewer: If we can remove this, then let's remove it before merging 😄 

```suggestion
```

**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases. At first, we were just a few contributors, but together we built a foundation for the amazing community that you see today. 🫂

We love that you — our community — have the freedom to work with our product in any way you want. Together we can test, experiment, and create a well-rounded product that covers a ton of different use cases. We're super proud of you 🙏

That's enough about us. Have fun. Be kind. Let's **Unleash** great software together. 💪

> > [📈 See our roadmap →](https://github.com/orgs/Unleash/projects/5)

Reviewer: Oh, wait, we actually created a new roadmap, so we should update that link too 🤓 Also, I suggest changing "see" to "check out" or similar for non-sighted users. 

```suggestion
> [📈 Check out our roadmap →](https://github.com/orgs/Unleash/projects/10)
```
